packages/core/src/controllers/SwapController.ts

```diff
@@ -702,14 +714,18 @@ export const SwapController = {
       ?.symbol} to ${NumberUtil.formatNumberToLocalString(toTokenAmount, 3)} ${state.toToken
       ?.symbol}`
 
-    RouterController.pushTransactionStack({
-      view: 'Account',
-      goBack: false,
-      onSuccess() {
-        SnackController.showLoading(snackbarPendingMessage)
-        SwapController.resetState()
-      }
-    })
+    if (isAuthConnector) {
+      RouterController.pushTransactionStack({
+        view: 'Account',
+        goBack: false,
+        onSuccess() {
+          SnackController.showLoading(snackbarPendingMessage)
+          SwapController.resetState()
+        }
+      })
+    } else {
+      SnackController.showLoading(snackbarPendingMessage)
```

Comment:
same

Reply:
Yes, this is just a feedback to the users that let them know they need to approve the TX, toast will be visible for few seconds.